ap.md

```diff
@@ -0,0 +1,256 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+# Build Your Own Internet
+</div>
+<div class="large-7 small-12 columns">
+
+As explained in [wifi tutorial](/wifi.html), Tessel 2 can easily connect to wifi networks using the command line tool. If we're using a device connected to the same network as the Tessel, then we can connect to it through our device's web browser. Like most network-connected devices, communication over the web depends on a shared router, or access point, to direct requests and responses between every device on the network or through the Internet. This is fine and dandy if we want to use Tessel in a place with an available network, but what happens when that's not possible? 
+
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/router.png)
+</div>
+<div class="large-7 small-12 columns">
+Instead of requiring a shared network through a standalone router, we can use the command line tool for configuring Tessel to emit a custom Wifi network and use it as an access point. Then, using Node, we can also start a web server for communicating with any connected device. Unfortunately, we won't be able to connect to the Internet but there is still a lot we can do without that. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.
```

Comment:
This is hard, trying to figure out how to concisely explain what this is. Nice job setting this up.

I started poking around to see if I could shorten and/or clarify and ended up moving a bunch of stuff around. I ended up with this:

```
# Build Your Own Internet

A typical WiFi setup consists of a couple of devices: a modem, which is connected by wire to the internet, and a wireless access point (also called a WAP or router), wired to the modem, which directs requests and responses between every device on the network.

As explained in [wifi tutorial](/wifi.html), Tessel 2 can easily connect to wifi networks using the command line tool. If your computer is connected to the same WAP as the Tessel, then the two devices can talk to each other. This is how we push code to Tessel over LAN (local area network).

The connection setup looks about like this:

![router](./images/router.png)

This is fine and dandy if we want to use Tessel in a place where there's a modem and WAP already set up, but what happens when that's not possible?

We can't access the Internet without a modem, but we can create a LAN connection between devices by turning Tessel 2 into a WAP. Devices on the network can all talk to each other, like this:

![router](./images/Tessel-AP.png)

This tutorial shows you to turn Tessel into a wireless access point, thus creating your own local network.

Part 2 of the tutorial shows how to start a web server which can communicate with device connected to the network.

By the end of this page, we'll be able to control the Tessel LEDs through a web app served by the Tessel.
```

Hopefully an improvement?